AI/BattleAI/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ set(battleAI_HEADERS
 		BattleExchangeVariant.h
 )
 
+find_package(TBB REQUIRED)
```

Comment:
it must be fixed here instead: https://github.com/vcmi/vcmi/blob/1b10aaa493e54947d6829eefd1673d462f5ec406/CMakeLists.txt#L522-L523